common/lib/xmodule/xmodule/raw_module.py

```diff
@@ -60,6 +59,14 @@ def definition_to_xml(self, resource_fs):
             raise SerializationError(self.location, msg)
 
 
+class RawDescriptor(RawMixin, XmlDescriptor, XMLEditingDescriptor):
```

Comment:
We move the methods to `RawMixin` which we `ProblemBlock` can inherit. The required code from `XmlDescriptor` and `XMLEditingDescriptor` is inherited separately keeping the MRO same.